tools/installers/install_s3prl.sh

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+MAKE=make
+
+if [ $# != 0 ]; then
+    echo "Usage: $0"
+    exit 1;
+fi
+
+torch_17_plus=$(python3 <<EOF
+from distutils.version import LooseVersion as V
+import torch
+
+if V(torch.__version__) >= V("1.7"):
+    print("true")
+else:
+    print("false")
+EOF
+)
+
+python_36_plus=$(python3 <<EOF
+from distutils.version import LooseVersion as V
+import sys
+
+if V(sys.version) >= V("3.6"):
+    print("true")
+else:
+    print("false")
+EOF
+)
+
+cuda_version=$(python3 <<EOF
+import torch
+if torch.cuda.is_available():
+    version=torch.version.cuda.split(".")
+    # 10.1.aa -> 101
+    print(version[0] + version[1])
+else:
+    print("")
+EOF
+)
+echo "cuda_version=${cuda_version}"
+
+if "${torch_17_plus}" && "${python_36_plus}"; then
+
+    rm -rf s3prl
+
+    # S3PRL Commit id when making this PR: `commit a6d2ae11c644c024cd096b3dc362f44dc91a51e5`
+    git clone https://github.com/s3prl/s3prl.git
```

Comment:
git clone https://github.com/s3prl/s3prl.git -b superb-interspeech2021

Reply:
It is better to add TODO comment to use tagged version?

Reply:
Good suggestion. Will do it.